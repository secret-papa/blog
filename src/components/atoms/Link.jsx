import styled from "@emotion/styled";
import { Link as GatsbyLink } from "gatsby";

const Link = styled(GatsbyLink)`
  text-decoration: none;
  color: inherit;
`;

export default Link;
