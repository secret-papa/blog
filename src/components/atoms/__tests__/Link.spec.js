import React from "react";
import { render } from "@testing-library/react";

import Link from "../Link";

describe("<Link />", () => {
  it("should show correctly", () => {
    const { container } = render(<Link to="." />);
    expect(container).toMatchSnapshot();
  });
});
