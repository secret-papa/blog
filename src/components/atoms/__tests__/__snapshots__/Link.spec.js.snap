// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should show correctly 1`] = `
.emotion-0 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

<div>
  <a
    class="emotion-0 emotion-1"
    href="."
  />
</div>
`;
